tests/ctl/api/test_datamap.py

```diff
@@ -544,6 +556,22 @@ def test_datamap_with_privacy_declaration(
         assert response.json() == expected_response_payload
 
 
+@pytest.mark.integration
+@pytest.mark.usefixtures("system_privacy_declarations")
+def test_datamap_with_privacy_declaration_with_name(
+    test_config: FidesConfig,
+    test_client: TestClient,
+) -> None:
+    response = test_client.get(
+        test_config.cli.server_url
+        + API_PREFIX
+        + "/datamap/default_organization?include_deprecated_columns=true",
```

Comment:
nit: if I'm being really picky, you should do another query with this set to `false` and confirm the deprecated columns are excluded